### PR TITLE
fix: [PIE-2147]: Changed empty state images of pipelines

### PR DESCRIPTION
**Pull request by goyalsonali**
Summary:

Changed empty state images of pipeline

 Jira Links:

https://harness.atlassian.net/browse/PIE-2147

Screenshots:

Screenshot 2022-02-21 at 1 44 57 PM
Screenshot 2022-02-21 at 1 45 05 PM
Screenshot 2022-02-21 at 1 45 11 PM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by goyalsonali** [comment]
retrigger jest

File: src/modules/70-pipeline/pages/pipelines/PipelinesPage.tsx [hunk 1]
```diff
@@ -162,9 +163,14 @@ const PipelinesPage: React.FC<CDPipelinesPageProps> = ({ mockData }) => {
   const isCDEnabled = (selectedProject?.modules && selectedProject.modules?.indexOf('CD') > -1) || false
   const isCIEnabled = (selectedProject?.modules && selectedProject.modules?.indexOf('CI') > -1) || false
   const isCIModule = module === 'ci'
+  const isCFModule = module === 'cf'
   const searchRef = useRef<ExpandingSearchInputHandle>({} as ExpandingSearchInputHandle)
   const { NG_NATIVE_HELM } = useFeatureFlags()
 
+  const emptyStagePipelineImage = (): string => {
```
Comment (devinder-harness) [inline src/modules/70-pipeline/pages/pipelines/PipelinesPage.tsx:170]: This is optional but for more readability,  we can rename `emptyStagePipelineImage` to `getEmptyStagePipelineImage`

**Review by devinder-harness — APPROVED** [review]

File: src/modules/70-pipeline/pages/pipelines/PipelinesPage.tsx (hunk 1 above)
Comment (ishantmehtaharness) [reply src/modules/70-pipeline/pages/pipelines/PipelinesPage.tsx:170]: Why is this a function?
We can directly make it a variable
`const emptyStagePipelineImage = isCIModule ? buildpipelineIllustration : isCFModule ? flagpipelineIllustration : pipelineIllustration`

File: src/modules/70-pipeline/pages/pipelines/PipelinesPage.tsx [hunk 2]
```diff
@@ -83,8 +83,9 @@ import { useFeatureFlags } from '@common/hooks/useFeatureFlag'
 import { PipelineGridView } from './views/PipelineGridView'
 import { PipelineListView } from './views/PipelineListView'
 import PipelineFilterForm from '../pipeline-deployment-list/PipelineFilterForm/PipelineFilterForm'
-import pipelineIllustration from './images/pipelines-illustration.svg'
-
+import pipelineIllustration from './images/deploypipeline-illustration.svg'
+import buildpipelineIllustration from './images/buildpipeline-illustration.svg'
+import flagpipelineIllustration from './images/flagpipeline-illustration.svg'
```
Comment (ishantmehtaharness) [inline src/modules/70-pipeline/pages/pipelines/PipelinesPage.tsx:88]: Can you please check if these below added 3 image files are not duplicates?

Comment (goyalsonali) [reply src/modules/70-pipeline/pages/pipelines/PipelinesPage.tsx:88]: These images are not duplicate as each image has different component coloured as per the module, like CD: Cog
FF: Flag,CI: Closed box.

File: src/modules/70-pipeline/pages/pipelines/PipelinesPage.tsx (hunk 1 above)
Comment (goyalsonali) [reply src/modules/70-pipeline/pages/pipelines/PipelinesPage.tsx:170]: Yes, definitely we can make it. I will change it to a variable.

**Review by vkbansal — APPROVED** [review]

**Review by ishantmehtaharness — APPROVED** [review]
Looks Good

**Pull request merged**